src/templates/export.js

```diff
@@ -236,6 +242,10 @@ module.exports = (router) => {
   if (router.get) {
     router.get('/export', (req, res, next) => {
       const options = hook.alter('exportOptions', {}, req, res);
+      if (options) {
+        options.extraFormFields = (req.query.extra && req.query.extra.split(',').filter((field) => !!field));
```

Comment:
Can we name this something other than ```extra```?  Really what we are doing here is including other fields. Maybe call it ```include```?